database/src/main/resources/db/migration/R__GTFS_query.sql

```diff
@@ -184,6 +184,22 @@ SELECT array_agg(x.id)
          ORDER BY "external-interface-description-id", created DESC) x
 $$ LANGUAGE SQL STABLE;
 
+-- New sproc for getting packages is need when calendar wants to show what kind of traffic was back
+-- in the day. We need to set detection_date to pick only those packages that were downloaded when detection
+-- was run.
```

Comment:
Part of the comments might look a bit out-of-place after merging the PR. Following might be more suited here:

-- Returns list of packages for service_id, which were the latest on detection_date. I.e. excludes those downloaded after detection_date.